app/src/NativeBase/Theme/StaTheme.tsx

```diff
@@ -121,7 +129,28 @@ const StaTheme = extendTheme({
         fontWeight: '600',
       },
     },
+    Button: {
+      baseStyle: {
+        rounded: 'full',
+        marginX: '1',
```

Comment:
I don't think buttons should have a default X margin set on them -- looking at the current designs in Figma, buttons like 'Volunteer Now' fill the available width of the box they're in.

`ScrollView` could have default padding/margin on it (have suggested that [here](https://github.com/Scottish-Tech-Army/Volunteer-app/pull/116/files#diff-7fb51506007e99f0f934fb7ad1138d3e29d6c6295c4386d2f8f5fefe54960f4eR233)) as screens should generally have the same horizontal spacing around them.

Reply:
Agree with Joe here, just to mention this button works well has have used it in the Project Details screen I've been updating. Small additional note here, I think the font in the design is white rather than the offwhite.